go.mod

```diff
@@ -22,6 +20,7 @@ require (
 	github.com/pmezard/go-difflib v1.0.0 // indirect
 	github.com/rogpeppe/go-internal v1.11.0 // indirect
 	github.com/spdx/tools-golang v0.5.3
+	github.com/spf13/cobra v1.7.0 // indirect
```

Comment:
@puerco i know its not this PR change but maybe some one can point out why are we bound to cobra in the lib?

Reply:
Ill take a look